jupyter_server/firewall.py

```diff
@@ -0,0 +1,75 @@
+from typing import Optional, Union
+
+from openapi_core import create_spec
+from tornado import httpclient, httputil
+from tornado.log import access_log
+from tornado_openapi3 import RequestValidator
+
+
+class FireWall:
```

Comment:
ha, yeah: this isn't really a firewall. we need something else to call it, i guess... 
- filter isn't bad 
- SpecPatch
- RoutePatch

hmmmm...

Reply:
- `SpecEnforcer`
- `SpecValidator`

Reply:
Let's go for `SpecValidator` as by itself it does not enforce anything.